### PR TITLE
Updating package.json and bootstrap.js for upcoming stimulus-bridge

### PR DESCRIPTION
**Pull request by weaverryan**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | It appears nothing is needed

This is waiting on:
* [x] Waiting on the 1.2.0 tags to make it to the sub-tree splits of the UX packages - e.g. https://github.com/symfony/ux-chartjs/tags
* [x] Merge of https://github.com/symfony/stimulus-bridge/pull/19
* [x] Release of symfony/stimulus-bridge
* [x] Release of @symfony/webpack-encore

This affects all version of WebpackEncoreBundle... but it's an odd case where the bundle and Webpack encore itself are not very "bound" together. We could update this recipe in only the latest WebpackEncoreBundle version... or even bump a new minor version of the bundle and add it there. This relates to https://github.com/symfony/recipes#updating-recipes

### CODE REVIEW

File: symfony/webpack-encore-bundle/1.0/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "devDependencies": {
-        "@symfony/stimulus-bridge": "^1.1.0",
-        "@symfony/webpack-encore": "^0.32.0",
+        "@symfony/stimulus-bridge": "^1.2.0",
+        "@symfony/webpack-encore": "^1.0.0",
```
Comment (OskarStark) [inline symfony/webpack-encore-bundle/1.0/package.json:4]: Isn't this considering a BC break if we now require a new major version? Or does it contain no breaks?

Comment (weaverryan) [reply symfony/webpack-encore-bundle/1.0/package.json:4]: Yes, the new version does contain BC breaks. But this is the awkward things with the Encore recipe: the bundle does not have or need a new version. But we will want new projects to begin using `@symfony/webpack-encore` 1.0.

**Review by fabpot — APPROVED** [review]

**Comment by jmsche** [comment]
Hi, is it possible to merge this? Bridge 2.0 was tagged yesterday, and so was UX 1.2.0, so it would help a lot for new installs.

**Comment by weaverryan** [comment]
I'm getting it merged now - the UX individual packages (despite the tag timestamp) weren't pushed until a few hours ago. I was waiting until then - but we're good now!

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**